### PR TITLE
EPMRPP-77854 || implement system message component

### PR DESCRIPTION
**Pull request by RuslanLyubimov**

### CODE REVIEW

**Review by AmsterGet — CHANGES_REQUESTED** [review]
Please leave only 1 commit related to the task.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/systemMessage/systemMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <p>{textSolution}</p>
+      </div>
+    </div>
+  );
+};
+
+SystemMessage.propTypes = {
+  message: PropTypes.string,
+  textSolution: PropTypes.string,
+  children: PropTypes.string,
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.jsx:37]: Please make children as a react node to have ability provide any type here

File: app/src/componentLibrary/systemMessage/systemMessage.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.jsx:1]: Please add standard license text here

File: app/src/componentLibrary/systemMessage/systemMessage.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import systemErrorImg from './img/SytemError.png';
+import warningImg from './img/warning.png';
+import styles from './systemMessage.scss';
+
+const cx = classNames.bind(styles);
+const MODE_INFORMATION = 'info';
+const MODE_WARNING = 'warning';
+const MODE_SYS_ERROR = 'error';
+let img = null;
+export const SystemMessage = ({ message, textSolution, children, mode }) => {
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.jsx:13]: According to the task component should have 2 states
1. width should be according to content
2. width should be 100% of parent container

Do you provide this behaviour?

File: app/src/componentLibrary/systemMessage/systemMessage.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.system-message-container {
+  min-height: 104px;
+  height: auto;
+  width: 100%;
+  background: $COLOR--bg-000;
+  border-radius: 8px;
+  font-family: $FONT-ROBOTO-REGULAR;
+  box-shadow: 0px 1px 3px rgba(55, 67, 98, 0.1);
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.scss:24]: Please extract box-shadow to corresponding .scss file in common variables with name from design

File: app/src/componentLibrary/systemMessage/systemMessage.scss [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  font-size: $size;
+  line-height: $height;
+  color: $color;
+}
+
+@mixin message-headers($color) {
+  color: $color;
+  text-transform: capitalize;
+  margin-bottom: 8px;
+  font-weight: 500;
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.scss:12]: font roboto with font-weight: 500 it is a $FONT-ROBOTO-MEDIUM
fw: 400 - $FONT-ROBOTO-REGULAR
fw: 500 - $FONT-ROBOTO-MEDIUM
fw: 700 - $FONT-ROBOTO-BOLD

File: app/src/componentLibrary/systemMessage/systemMessage.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+  .text-container {
+    padding: 12px 16px 16px;
+
+    .message-header-info {
+      @include message-headers($COLOR--e-400);
+    }
+
+    .message-header-warning {
+      @include message-headers($COLOR--warning-message);
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.scss:41]: I cannot find naming for this color in DS pls discuss it with UX

File: app/src/componentLibrary/systemMessage/systemMessage.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
+@mixin font-config($weight, $size, $height, $color) {
```
Comment (chivekrodis) [inline app/src/componentLibrary/systemMessage/systemMessage.scss:1]: Please add standard license text here

**Review by chivekrodis — CHANGES_REQUESTED** [review]

File: app/src/common/css/variables/newColors.scss [hunk 8]
```diff
@@ -54,3 +54,5 @@ $COLOR--tag-value-text: #394db6;
 $COLOR--tag-value-background: #ced8fc;
 $COLOR--tag-key-text: #6f4599;
 $COLOR--tag-key-background: #dac3e6;
+$COLOR--warning-message: #d78706;
+$COLOR---system-error-message: #db3549;
```
[the remark this answers is not in the document]
Comment (RuslanLyubimov) [reply app/src/common/css/variables/newColors.scss:58]: Asked UX team, it will be corrected soon

**Comment by AmsterGet** [comment]
Closed in favor of https://github.com/reportportal/service-ui/pull/3171

**Pull request closed without merging**